benches/misc.rs

```diff
@@ -36,6 +36,30 @@ fn misc_gen_bool_var(b: &mut Bencher) {
     })
 }
 
+#[bench]
+fn misc_gen_ratio_const(b: &mut Bencher) {
+    let mut rng = StdRng::from_rng(&mut thread_rng()).unwrap();
+    b.iter(|| {
+        let mut accum = true;
+        for _ in 0..::RAND_BENCH_N {
+            accum ^= rng.gen_ratio(2, 3);
+        }
+        accum
+    })
+}
+
+#[bench]
+fn misc_gen_ratio_var(b: &mut Bencher) {
+    let mut rng = StdRng::from_rng(&mut thread_rng()).unwrap();
+    b.iter(|| {
+        let mut accum = true;
+        for i in 2..(::RAND_BENCH_N as u32 + 2) {
+            accum ^= rng.gen_ratio(1, i);
```

Comment:
Good benchmark :+1:. In theory the multiplication by the numerator can be optimized out. What do you think of
```rust
        for i in 1..(::RAND_BENCH_N as u32 + 1) {
            accum ^= rng.gen_ratio(i, i + 1);
```